lib/Subscriber/Core/RegistratorSubscriber.php

```diff
@@ -83,7 +84,7 @@ public function configureOptions(ConfigureOptionsEvent $event)
      *
      * @param HydrateEvent $event
      */
-    public function handleDefaultLocale(HydrateEvent $event)
+    public function handleDefaultLocale(AbstractMappingEvent $event)
```

Comment:
This is where you need a change manager to act upon the
MappingSubscriber (which SHOULD be where all the mapping happens) so as
to only write what has not changed.

However that would have only a marginal effect, the big issue would come
from the legact content system which needs to be decoupled from the
document manager as previously discussed.

On Tue, Apr 19, 2016 at 08:06:19AM -0700, Daniel Rotter wrote:
>    In [1]lib/Subscriber/Core/RegistratorSubscriber.php:
> 
>  > @@ -83,7 +84,7 @@ public function configureOptions(ConfigureOptionsEvent $event)
>  >       *
>  >       * @param HydrateEvent $event
>  >       */
>  > -    public function handleDefaultLocale(HydrateEvent $event)
>  > +    public function handleDefaultLocale(AbstractMappingEvent $event)
> 
>    Not the people concerned about performance, since an entire language es
>    saved, when it is not required at all.
> 
>    —
>    You are receiving this because you authored the thread.
>    Reply to this email directly or [2]view it on GitHub
> 
>    Reverse link: [3]unknown
> 
> References
> 
>    Visible links
>    1. https://github.com/sulu/sulu-document-manager/pull/72#discussion_r60248603
>    2. https://github.com/sulu/sulu-document-manager/pull/72/files/8e599707818caabeeea23d162b9a19cd78a0b234#r60248603
>    3. https://github.com/sulu/sulu-document-manager/pull/72/files/8e599707818caabeeea23d162b9a19cd78a0b234#r60248603